sass/style.css

```diff
@@ -317,6 +317,29 @@ h3.city {
   opacity: 0.4;
 }
 
+/*********************
+  * PARTNERS STYLES *
+  *********************/
+h1.page-title {
+  text-transform: uppercase;
```

Comment:
Did we vet this for other languages? Not sure if we want text-transform for all languages / countries